dash_bio/component_factory/_manhattan.py

```diff
@@ -1,54 +1,17 @@
 from __future__ import absolute_import
 
+import numpy as np
 import pandas as pd
 from pandas.api.types import is_numeric_dtype
-import numpy as np
 
 import plotly.graph_objs as go
 
+from ._volcano import _get_hover_text
```

Comment:
I'm a bit conflicted about leaving this in here, since the two plots don't really have anything to do with each other besides this shared functionality. If they're importing it from somewhere, it should come from a third file (in my opinion).

Reply:
How about moving this helper function to the `dash-bio-utils` package?

Reply:
We shouldn't do that, unless we're willing to include that package as a dependency for this package (which is the opposite of what we wanted). It's necessary for the *components* to work; the `dash-bio-utils` package is only necessary for the particular set of demo apps that we've created in this repo.

Reply:
Right, right, that's not the spirit. How about creating a `utils.py` module in here (`dash_bio/component_factory/`) then?

Reply:
I think that makes the most sense!

Reply:
Anyway, I would argue that Manhattan plots and Volcano plots do have something to do with each other: they can both make use of a hover text utility displaying 'SNP' and 'GENE' stuff so, apparently, evidently, clearly, they are used to represent a certain kind of datasets, i.e., those datasets with 'SNP' and 'GENE' data (which is already a lot to have in common).

The docstring starts by stating that it applies to both Manhattan and Volcano plots (already in the original submission, this is not something that I changed).

Reply:
Done via 6675dc30bc69441e165f00f5a6468e7fd7431d3a.